src/tests/test_grpc.py

```diff
@@ -228,8 +228,8 @@ def test_fetch_sequences_chromosomal(self, multi_dbs, genome_uuid, assembly_acce
             assembly_accession=assembly_accession,
             chromosomal_only=chromosomal_only
         )
-        logger.debug(f"Retrieved {test[0:2]}")
-        assert test[-1].AssemblySequence.chromosomal == expected_output
+        logger.debug(f"Retrieved {test[0]}")
+        assert test[0].AssemblySequence.chromosomal == expected_output
```

Comment:
I think this test is failing
```
  # Chromosomal and non-chromosomal
  ("3704ceb1-948d-11ec-a39d-005056b38ce3", "GCA_000001405.14", False, 0),
```
Because it's fetching both `chromosomal` and `non-chromosomal` sequences, and the first element `test[0]` is chromosomal (the last one `test[-1]` is not).

Reply:
Locally it's the other way around .... weird. I'll try to fix this for good (both local and travis tests passing)